src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    String color;
+    int number;

```

Comment:
Classes, fields, constructors, methods must have access modifiers otherwise default will be used that isn't a good practice. You should specify the access level explicitly, for example, private or protected, depending on the intended use.